src/database/bplus/mod.rs

```diff
@@ -0,0 +1,102 @@
+//! # Bplus Database [WIP]
+//!
+//! This module provides a database implementation backed by a
+//! persistent bplus tree.
+//!
+//! #### Authorship
+//!
+//! - Benjamin Riley Zimmerman, 9/11/2024 (bz931@berkeley.edu)
+
+/* IMPORTS */
+
+use std::path::Path;
+
+use anyhow::Result;
+use bitvec::prelude::{BitSlice, Msb0};
+
+#[allow(unused_imports)]
+use crate::{
+    model::State,
+    database::{Persistence, KVStore, Persistent, Tabular, Record, Schema, SchemaBuilder},
+};
+
+/* CONSTANTS */
+
+
+
+/* DATABASE DEFINITION */
+
+#[allow(dead_code)]
```

Comment:
don't worry about merging code with warnings -- it is preferrable to leave them, because it's easy to lose dead code once you allow it (it's better to have it scream at us so we remember to fix it later)

Reply:
sg, removed directives